test/e2e/crdvalidator_test.go

```diff
@@ -221,5 +224,141 @@ var _ = Describe("crd validation webhook", func() {
 				}).Should(ContainSubstring("error validating existing CRs against new CRD's schema"))
 			})
 		})
+
+		When("an incoming crd event is owned by a RukPak BundleInstance", func() {
+			var (
+				crd    *apiextensionsv1.CustomResourceDefinition
+				bundle *rukpakv1alpha1.Bundle
+				bi     *rukpakv1alpha1.BundleInstance
+			)
+			BeforeEach(func() {
+				// Build a bundle and bundleinstance for usage as an OwnerReference in the CRD to be tested
+				bundle = &rukpakv1alpha1.Bundle{
+					ObjectMeta: metav1.ObjectMeta{
+						GenerateName: "olm-crds",
+					},
+					Spec: rukpakv1alpha1.BundleSpec{
+						ProvisionerClassName: plainProvisionerID,
+						Source: rukpakv1alpha1.BundleSource{
+							Type: rukpakv1alpha1.SourceTypeImage,
+							Image: &rukpakv1alpha1.ImageSource{
+								Ref: "testdata/bundles/plain-v0:valid",
+							},
+						},
+					},
+				}
+				Eventually(func() error {
+					return c.Create(ctx, bundle)
+				}).Should(Succeed(), "should be able to create a safe bundle but was not")
+
+				bi = &rukpakv1alpha1.BundleInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						GenerateName: "olm-crds",
+					},
+					Spec: rukpakv1alpha1.BundleInstanceSpec{
+						ProvisionerClassName: plainProvisionerID,
+						BundleName:           bundle.GetName(),
+					},
+				}
+				Eventually(func() error {
+					return c.Create(ctx, bi)
+				}).Should(Succeed(), "should be able to create a safe bundleinstance but was not")
+
+				// Build the CRD and add the BundleInstance's OwnerReference values to it
+				crd = testutil.NewTestingCRD("", testutil.DefaultGroup, map[string]string{},
+					[]apiextensionsv1.CustomResourceDefinitionVersion{{
+						Name:    "v1alpha1",
+						Served:  true,
+						Storage: true,
+						Schema: &apiextensionsv1.CustomResourceValidation{
+							OpenAPIV3Schema: &apiextensionsv1.JSONSchemaProps{
+								Type:        "object",
+								Description: "my crd schema",
+								Properties: map[string]apiextensionsv1.JSONSchemaProps{
+									"sampleProperty": {Type: "string"},
+								},
+							},
+						},
+					}},
+				)
+				crd.OwnerReferences = testutil.NewRukpakOwnerReference(bi.GetName(), bi.Kind, bi.GetUID())
+				Eventually(func() error {
+					return c.Create(ctx, crd)
+				}).Should(Succeed(), "should be able to create a safe crd but was not")
```

Comment:
This shouldn't be necessary as we should be placing owner references on all underlying Bundle source contents.

Reply:
Updated the most recent tests to a place where this is no longer an issue.